src/features/offer/pages/tests/Offer.analytics.test.tsx

```diff
@@ -81,6 +94,7 @@ describe('<Offer /> - Analytics', () => {
       // 1st scroll to bottom => trigger
       await scrollView.props.onScroll({ nativeEvent: nativeEventBottom })
     })
+    await superFlushWithAct(25)
     expect(analytics.logConsultWholeOffer).toHaveBeenCalledWith(offerId)
```

Comment:
Je te conseille d'essayer avant de faire un `superFlushWithAct` : 
```suggestion
    await waitForExpect(() => { 
        expect(analytics.logConsultWholeOffer).toHaveBeenCalledWith(offerId)
    })
```
Si cela ne marche, alors go pour le superFlushWithAct.